spec/features/users_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'rails_helper'
+
+feature 'Signing up' do
+
+  scenario 'users can sign up to start using travelgram' do
+    visit '/'
+    sign_up(username: 'allimac',email: 'camilla@email.com', password: 'pass1234')
+    expect(page).to have_content 'Welcome!'
+    expect(page).to have_css('img', 'user.png')
+    expect(current_path).to eq '/'
+  end
+
+end
+
+feature 'Logging in' do
+
+  background do
+    visit '/'
+    sign_up(username: 'allimac',email: 'camilla@email.com', password: 'pass1234')
```

Comment:
Metrics/LineLength: Line is too long. [81/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)